packages/optimizer/spec/end-to-end/body-only/expected_output.default.html

```diff
@@ -6,6 +6,7 @@
   <link rel="preload" href="https://cdn.ampproject.org/v0.js" as="script">
   <script data-auto async src="https://cdn.ampproject.org/v0.js"></script>
   <script async src="https://cdn.ampproject.org/v0/amp-video-0.1.js" custom-element="amp-video"></script>
+  <link rel="preload" href="https://amp.dev/static/samples/img/tokyo.jpg" as="image" data-hero>
```

Comment:
just for my own edification - what is the purpose of the data attr?

Reply:
For being able to track which images have been identified as hero images